mockrxandroidble/src/main/java/com/polidea/rxandroidble/mockrxandroidble/RxBleDeviceMock.java

```diff
@@ -76,6 +76,12 @@ public void addAdvertisedUUID(UUID advertisedUUID) {
     }
 
     @Override
+    public BluetoothDevice getBluetoothDevice() {
```

Comment:
Yes, for pull requests these reorderings only make it worse to read. For refactorings that are not changing code but only reordering - you can push directly.